mu4e/mu4e-message.el

```diff
@@ -181,6 +182,17 @@ representation."
       ;; otherwise, use text
       (t nil))))
 
+(defun mu4e~want-format-flowed (msg)
```

Comment:
> furthermore, seems that it's a property of a specific mime-part, not necessarily of the message as a whole, or?

Yeah, as I mentioned in the PR description, I fear this approach is too simplistic because of the way mu accumulates plain-text bodies. It could produce very nasty results if some plain-text bodies are marked format=flowed and others are not, but then everything is format-flowed. Hence I submitted this as a starting point for further discussion rather than expecting it to be merged in its current form.